builder/builder.go

```diff
@@ -194,7 +194,7 @@ mix-bundles/`
 
 // InitMix will initialise a new swupd-client consumable "mix" with the given
 // based Clear Linux version and specified mix version.
-func (b *Builder) InitMix(upstreamVer string, mixVer string, allLocal bool, allUpstream bool, upstreamURL string, git bool) error {
+func (b *Builder) InitMix(upstreamVer string, mixVer string, allLocal bool, allUpstream bool, noDefaults bool, upstreamURL string, git bool) error {
```

Comment:
Eww another bool to the function signature. Maybe this should take a struct with all the bools that are needed? This API is very difficult to use...

/me channels his inner @icarus-sparry

Reply:
Yeah, it's gross. But it was already gross before this addition. That's a bigger redesign than I want to do in a small feature PR like this.

Reply:
fiiiiiiiiine :)

Reply:
The question ought to be "how many of the 16 possible combinations of bool make sense" and unless it is "most" then this ought to be cleaned up. Fair enough not to do it in this PR.

 If you are in an environment where you have say godoc support then it is sort of OK, but if all you see is function(T,T,F,T) then it is hard to reason about what the 3 true and one false is selecting.